frontend/public/components/environment.jsx

```diff
@@ -80,6 +80,149 @@ const stateToProps = ({k8s}, {obj}) => ({
   model: k8s.getIn(['RESOURCES', 'models', referenceFor(obj)]) || k8s.getIn(['RESOURCES', 'models', obj.kind]),
 });
 
+class CurrentEnvVars {
+  constructor() {
+    this.currentEnvVars = {};
+  }
+
+  instantiateWithRawData(rawEnvData) {
+    this.isContainerArray = _.isArray(rawEnvData.containers);
+    this.hasInitContainers = !_.isUndefined(rawEnvData.initContainers);
```

Comment:
```js
this.hasInitContainers = !_.isEmpty(rawEnvData.initContainers);
```

Reply:
Instead of having a separate property to track this, can we simple check `_.isEmpty(this.currentEnvVars.initContainers)` when we need to? Then there's no danger of this value getting out of sync.

Reply:
Yeah I added this to prevent what you said...I figured constantly checking is repetitive, and just keep it in one place instead. It won't be out of sync.

Reply:
This won't work, but I can add a method instead if you prefer.